test/functional/legacy/writefile_spec.lua

```diff
@@ -4,7 +4,7 @@ local helpers = require('test.functional.helpers')(after_each)
 local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
 
 describe('writefile', function()
-  setup(clear)
+  before_each(clear)
```

Comment:
Are you positive that change is warranted? If you used `legacy2luatest.pl`, this is a modification someone else made after that, so chances are it should stay (unless you understood that and made that change because you figured it was neccessary, in which case disregard what I said).